### DISCUSSION TITLE
Use an env specific teamsapp.env.yml file with Teams Toolkit CLI

### DISCUSSION THREAD

**Discussion by Benjiiim** [OfficeDev/microsoft-365-agents-toolkit#12550] [Q&A] [answered]
On top of teamsapp.yml, teamsapp.local.yml and teamsapp.testtool.yml, I would need an other project file for a specific environment.
For example, a teamsapp.prod.yml that would be call through teamsapp provision, teamsapp deploy or teamsapp publish commands when using the --env prod parameter.
According to the [Teams Toolkit CLI doc](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/teams-toolkit-cli), that is not possible.
Am I missing something?

**Comment by therealjohn** [comment, marked as answer]
Those are the only three YML files supported. 

teamsapp.local.yml is for the 'local' environment, that is either using `--env local` with the CLI or F5/Start Debugging in VS or VS Code.

teamsapp.yml is for _all_ other custom environments, intended for 'remote' hosting in a place like Azure. So `prod`, `dev`, `{name}` would all use that configuration. 

teamsapp.testtool.yml is another special one like local except unique to the App Test Tool debug target where most provisioning pieces are not needed, but you might want to run a script or NPM commands as part of the F5/Start Debugging experience.
